New plugin version

mateuszpy opened the pull request:
Old plugin version (1.4.0) isn't working with new PrestaShop versions so here is updated one with fixes necessary to run on PrestaShop 1.6.1.0. Plugin has been checked using validator at http://validator.prestashop.com

mateuszpy committed:
Merge pull request #1 from dotpay/Beta

Beta

mateuszpy committed:
Merge pull request #2 from dotpay/Beta

Beta

Quetzacoalt91 commented on dotpay.php:
```diff
@@ -82,6 +82,8 @@ public function install()
 	
         Configuration::updateValue('DP_TEST', true);
+        Configuration::updateValue('DP_CHK', false);
+        Configuration::updateValue('DP_SSL', false);
```
Maybe you could add these configuration keys in an upgrade file, or these configuration values won't exist if the module is already installed..

Quetzacoalt91 commented:
Hi @mateuszpy,

Why did you duplicate payment_return.tpl in /front and /hook folders in this pull-request ?

mateuszpy committed:
v1.4.9

-deleted views\templates\front\payment_return.tpl
-deleted ID and PIN values in dotpay.php
-added upgrade\upgrade-1.4.9.php

mateuszpy commented:
We addressed both of your comments and made some small changes in the module.
Duplicate was made for compatibility some time ago but it's not needed anymore.

Quetzacoalt91 commented:
Hi @mateuszpy,

Thanks, your PR will be merged on Monday.

mateuszpy commented:
Thanks for the information @Quetzacoalt91 
Does it mean updated plugin is going to appear on the list of PrestaShop modules on Monday?

mateuszpy committed:
1.4.9

-translation improvements
-changes in payment.tpl and payment return.tpl

Quetzacoalt91 commented:
Hi @mateuszpy,

Thanks, your PR has just been merged and the changes will be available from tomorrow.

Best regards

mateuszpy commented:
Thank you @Quetzacoalt91
